### PR TITLE
Namespace mapping to database in observer in one HBase client; set RPC connect timeout to Table Client

### PR DESCRIPTION
**Pull request by JackShi148**
## Summary

One Hbase client can map different namespaces to corresponding databases in oceanbase. Add RPC connection timeout parameter in our Configuration.

## Solution Description

Use the same format of namespaces and qualifiers to map to our databases and tables. Like "n1:test" will map to database "n1" and table group "n1:test". The family will become the suffix to form a complete table name in oceanbase.

### CODE REVIEW

File: src/test/java/com/alipay/oceanbase/hbase/HTableTestBase.java [hunk 1]
```diff
@@ -3579,6 +3579,7 @@ public void testGetColumnFamilyNotExists() throws Exception {
         get.addFamily("family_not_exists".getBytes());
         expectedException.expect(IOException.class);
         expectedException.expectMessage("query table:test family family_not_exists error.");
+        // expectedException.expectMessage("query table:n1:test family family_not_exists error.");
```
Comment (shenyunlong) [inline src/test/java/com/alipay/oceanbase/hbase/HTableTestBase.java:3582]: remove useless comments

File: src/test/java/com/alipay/oceanbase/hbase/OHTableClientTest.java [hunk 2]
```diff
@@ -57,7 +59,9 @@ public void testNew() throws Exception {
         hTable2.setWriteBufferSize(10000000L);
         assertEquals(10000000L, hTable2.getWriteBufferSize());
         assertEquals("test", hTable2.getTableNameString());
+        //        assertEquals("n1:test", hTable2.getTableNameString());
         assertEquals("test", new String(hTable2.getTableName()));
+        //        assertEquals("n1:test", new String(hTable2.getTableName()));
```
Comment (shenyunlong) [inline src/test/java/com/alipay/oceanbase/hbase/OHTableClientTest.java:64]: need confirm this comments

File: src/test/java/com/alipay/oceanbase/hbase/HTableTestBase.java (hunk 1 above)
Comment (JackShi148) [reply src/test/java/com/alipay/oceanbase/hbase/HTableTestBase.java]: Already removed.

File: src/test/java/com/alipay/oceanbase/hbase/OHTableClientTest.java (hunk 2 above)
Comment (JackShi148) [reply src/test/java/com/alipay/oceanbase/hbase/OHTableClientTest.java:64]: Already removed. Add a new test case file for multi-namespace testing.

**Comment by shenyunlong** [comment]
LGTM

**Pull request merged**